code/script.js

```diff
@@ -1,51 +1,252 @@
+const nameForm = document.getElementById('name-form');
+//const emailForm = document.getElementById('email-form');
+const nameInput = document.getElementById('name-input');
+const sendBtn = document.getElementById('send-btn');//
+const inputWrapper = document.getElementById('input-wrapper');
+//const chatToggle = document.getElementById('js-chat-toggle');
+//const chatContainer = document.getElementById('js-chat-container')
+
+const chatToggle = document.getElementById('js-chat-toggle');
+const chatContainer = document.getElementById('js-chat-container')
 
 // If you need any global variables that you can use across different functions, declare them here:
 
+//let userEmail = '';
+let userName = '';
+const choice = '';
+const yourPrice = '';
 
 // Declare your functions after this comment
 
-// This function will add a chat bubble in the correct place based on who the sender is
+// This function will add a chat bubble in the correct place based on who the sender is. I think?
 const showMessage = (message, sender) => {
-  // the if statement checks if the sender is 'user' and if that's the case it inserts an html senction inside the chat with the posted message
+
   if (sender === 'user') {
     chat.innerHTML += `
       <section class="user-msg">
         <div class="bubble user-bubble">
           <p>${message}</p>
         </div>
-        <img src="assets/user.png" alt="User" />  
+        <img src="assets/user1.png" alt="User" />  
       </section>
     `
-    // the else if statement checks if the sender is a bot and if that's the case it inserts an html senction inside the chat with the posted message
-  } else if (sender === 'bot') {
+  }
+  else if (sender === 'bot') {
+    console.log(sender);
+    console.log(message);
     chat.innerHTML += `
       <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
+        <img src="assets/bot1.png" alt="Bot" />
         <div class="bubble bot-bubble">
           <p>${message}</p>
         </div>
-      </section>
-    `
+      </section>`
   }
   // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
   chat.scrollTop = chat.scrollHeight
 }
 
 // Starts here
+// here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
+
 const greetUser = () => {
-  // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, What's your name?", 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  showMessage(`Hello there!`, 'bot')
+  showMessage(`I'm Anna, a chatbot.`, 'bot')
+  showMessage(`What's your name?`, 'bot')
+}
+
+const userNameInput = event => {
+  event.preventDefault();
+  //I think the name is saved for later, but is it saved globally?
```

Comment:
I think Yes. When you declare a variable in the global context, then it will be globally available.